Ladybird/Qt/Tab.cpp

```diff
@@ -893,15 +894,28 @@ void Tab::resizeEvent(QResizeEvent* event)
 
 void Tab::update_hover_label()
 {
+    m_hover_label->setText(QFontMetrics(m_hover_label->font()).elidedText(m_hover_label->text(), Qt::ElideRight, width() / 2));
     m_hover_label->resize(QFontMetrics(m_hover_label->font()).boundingRect(m_hover_label->text()).adjusted(-4, -2, 4, 2).size());
-    auto hover_label_height = height() - m_hover_label->height() - 8;
+
+    auto hover_label_height = height() - m_hover_label->height();
     if (m_find_in_page->isVisible())
-        hover_label_height -= m_find_in_page->height() - 4;
+        hover_label_height -= m_find_in_page->height() - 8;
```

Comment:
This causes the label and the find in page widget to overlap on my machine:

![image](https://github.com/user-attachments/assets/56ca89f4-6307-4583-bc45-efccfd4fb5f0)